test_infra/src/lib.rs

```diff
@@ -798,6 +798,17 @@ pub fn check_matched_lines_count(input: &str, keywords: Vec<&str>, line_count: u
     }
 }
 
```

Comment:
There is some kind of wrapping issue in your commit message.

Reply:
I don't know what the meaning. 😕